api/api_test.go

```diff
@@ -139,15 +139,13 @@ func TestNoManifestGivesError(t *testing.T) {
 	assert.Contains(t, string(rr.Body.Bytes()), manifestUrl)
 }
 
-func TestValidDeploymentRequestAndManifestCreateResources(t *testing.T) {
+//TODO remove once grace period ends
+func TestWarningsWhenUsingOldPropertyNames(t *testing.T) {
```

Comment:
Better to just specify that the test is deprecated.
`// Deprecated`, gives you a nice ~~~line~~~.

Reply:
the test is not deprecated but will die once the deprecation is effectuated